src/components/message/message.twig

```diff
@@ -0,0 +1,23 @@
+{# 
+  @TODO
+  1. Generate an example of a message with a decorative icon.
+  #}
+
+{% set tagName = tagName | default('div') %}
+
+<{{tagName}} aria-describedby="MessageInfo" {% if not stopAnnouncements %} aria-live="polite"{% endif %} class="c-message{% if class %} {{class}}{% endif
+%}" tabindex=”0”{% if isWarning %} role="alert"{% else %} role="status"{% endif
+%}>
```

Comment:
- I worry that these attributes will slow down approval of the basic CSS/markup pattern because they are so intertwined with how this pattern may be used in context. Is there any way we can simplify for now?
- I see some smart quotes where dumb quotes should be.

Reply:
@tylersticka 

> I worry that these attributes will slow down approval of the basic CSS/markup pattern because they are so intertwined with how this pattern may be used in context.

> Is there any way we can simplify for now?

I see two options: 

1. I could remove all accessibility related attributes
2. I could remove some of the attributes. 

If the later, I'm not sure which ones should be considered default. 

@tylersticka

Reply:
Let's remove all of them for now.